src/usecases/tos/product.js

```diff
@@ -0,0 +1,4 @@
+import { config } from '@/config'
+
+export default productId =>
+  `${config.get('etvasURL')}/terms-and-conditions/${productId}`
```

Comment:
isn't this something like `https://dz-bank.helloetvas-dev.xyz/products/key-depot/28646d37-7a52-4115-a85c-250f162933ff/terms-and-conditions` ?

Reply:
The URL's starting with `/products/*` are for authenticated flows. For other flows, there are prefixes (like `embed` for embedding parts or `l` for landing pages).

The unauthenticated (publicly available) URL's tend to be exposed and visibly provided to our partners, so we want to keep them as short as possible.

The Etvas Platform T&C's are available at `/terms-and-conditions` so, if you append a product id to that URL, it will provide the product T&C.